Stripe/STPAspects.m

```diff
@@ -175,10 +175,14 @@ static BOOL stp_aspect_remove(STPAspectIdentifier *aspect, NSError * __autorelea
 }
 
 static void stp_aspect_performLocked(dispatch_block_t block) {
-    static OSSpinLock aspect_lock = OS_SPINLOCK_INIT;
-    OSSpinLockLock(&aspect_lock);
+    static NSLock *aspectLock = nil;
+    static dispatch_once_t onceToken;
```

Comment:
I used the suggested `os_unfair_lock` in my patch, which seems to be a more direct replacement for OSSpinLock. You can take a look once I merge this patch into mine.